web/api/v1/api.go

```diff
@@ -108,6 +108,31 @@ type rulesRetriever interface {
 	AlertingRules() []*rules.AlertingRule
 }
 
+// PrometheusVersion contains build information about Prometheus.
+type PrometheusVersion struct {
+	Version   string `json:"version"`
+	Revision  string `json:"revision"`
+	Branch    string `json:"branch"`
+	BuildUser string `json:"buildUser"`
+	BuildDate string `json:"buildDate"`
+	GoVersion string `json:"goVersion"`
+}
+
+type RuntimeInfo struct {
+	Birth               time.Time
```

Comment:
`Birth` was always a bit funny term, initially from 2013 :) How about we rename it to `StartTime` in the API?